src/test/java/io/vlingo/wire/fdx/bidirectional/netty/client/NettyClientRequestResponseChannelTest.java

```diff
@@ -1,189 +1,189 @@
-//    final ServerBootstrap b = new ServerBootstrap();
-//
-//    return b.group(parentGroup, childGroup)
-//            .channel(NioServerSocketChannel.class)
-//            .childHandler(new ChannelInitializer<SocketChannel>() {
-//              @Override
-//              public void initChannel(SocketChannel ch) throws Exception {
-//                ch.pipeline()
-//                  .addLast(new ChannelInboundHandlerAdapter() {
-//                    private byte[] partialRequestBytes;
-//
-//                    @Override
-//                    public void channelActive(final ChannelHandlerContext ctx) throws Exception {
-//                      super.channelActive(ctx);
-//                      connectionsCount.countDown();
-//                    }
-//
-//                    @Override
-//                    public void channelRead(final ChannelHandlerContext ctx, final Object msg) throws Exception {
-//                      final ByteBuf byteBuf = (ByteBuf) msg;
-//
-//                      while (byteBuf.readableBytes() >= requestMsgSize) {
-//                        final String request;
-//                        if (partialRequestBytes != null && partialRequestBytes.length > 0) {
-//                          final int remainingBytes = requestMsgSize - partialRequestBytes.length;
-//                          request = new String(partialRequestBytes) + byteBuf.readCharSequence(remainingBytes, Charset.defaultCharset())
-//                                                                             .toString();
-//                          partialRequestBytes = null;
-//                        } else {
-//                          request = byteBuf.readCharSequence(requestMsgSize, Charset.defaultCharset())
-//                                           .toString();
-//                        }
-//
-//                        serverReceivedMessagesCount.countDown();
-//                        serverReceivedMessage.add(request);
-//
-//                        final String reply = request + " reply";
-//
-//                        serverSentMessages.add(reply);
-//
-//                        final ByteBuf replyBuffer = ctx.alloc()
-//                                                       .buffer(reply.getBytes().length);
-//
-//                        replyBuffer.writeBytes(reply.getBytes());
-//
-//                        ctx.writeAndFlush(replyBuffer);
-//                      }
-//
-//                      if (byteBuf.readableBytes() > 0) {
-//                        partialRequestBytes = new byte[byteBuf.readableBytes()];
-//                        byteBuf.readBytes(partialRequestBytes);
-//                      }
-//                    }
-//                  });
-//              }
-//            })
-//            .bind(testPort)
-//            .sync()
-//            .await();
-//  }
-//
-//}
\ No newline at end of file
+// Copyright © 2012-2020 VLINGO LABS. All rights reserved.
```

Comment:
@rosetau Thanks. Is it possible to eliminate the changes to this test altogether from the PR so nothing is actually changed?

Reply:
In #3 issue `TestResponseChannelConsumer` needed to be refactored, so I did that. At the same time, `NettyClientRequestResponseChannelTest` uses `TestResponseChannelConsumer`' `AccessSafely` member.
Changes to `Netty...Test` could be reverted with leaving `TestResponseChannelConsumer` using `Test Until` only, which contradicts #3 goal.

Reply:
@rosetau Thanks for explaining. I am merging with all your changes. Thanks for your help!

Reply:
@VaughnVernon Thank you for merging! Shouldn't #3 be closed now?